### PR TITLE
Initial German translation commit

### PR DESCRIPTION
**Pull request by flaymeariver**
Added files with the initial German translations of the given variables

### CODE REVIEW

File: de-DE/Common.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  'Navigation': {
+    'Sessions': 'Sessions',
+    'Server': 'Server Einstellungen',
+    'Players': 'Spieler + Team',
+    'Badges': 'Orden',
+    'Pokédex': 'Pokédex',
+    'Web': 'Web Sources',
+    'Nuzlocke': 'Nuzlocke Tools',
+    'Settings': 'Einstellungen',
+    'About': 'About'
```
Comment (NoelDavies) [inline de-DE/Common.js:11]: Are the following correct German translations?

- Sessions
- Web Sources
- "Tools"
- About
-

File: de-DE/Common.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export default {
+  'Navigation': {
```
Comment (NoelDavies) [inline de-DE/Common.js]: Are the following correct German translations?

- Sessions
- Web Sources
- "Tools"
- About
-

File: de-DE/Settings.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  'Titles': {
+    'Session': 'Session-Einstellungen',
+    'System': 'System-Einstellungen',
+    'Window': 'Fenstereinstellungen',
+    'Accessibility': 'Einstellungen zur Barrierefreiheit',
+    'GameAndEmulator': 'Spiel- & Emulator-Einstellungen'
+  },
+  'Fields': {
+    'SessionName': {
+      'Titles': 'Session-Name',
```
Comment (NoelDavies) [inline de-DE/Settings.js]: Is this really hyphenated in German?

File: de-DE/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+const modules = {}
+
+files.keys().forEach(key => {
+  if (key === './index.js') return
+  modules[key.replace(/(\.\/|\.js)/g, '')] = files(key).default
+})
+
+export default {
+  label: 'Deutsch (DE)',
+  code: 'de-DE',
```
Comment (NoelDavies) [inline de-DE/index.js]: This should just be "de"

**Review by NoelDavies — COMMENTED** [review]
Can you rename this folder to just `de`?

File: de-DE/Common.js (hunk 2 above)
Comment (flaymeariver) [reply de-DE/Common.js]: These words are regularly used as so-called anglicisms in the German language in this context. There are existing German translations, but those would sound strange even to native speakers as myself.

File: de-DE/Settings.js (hunk 3 above)
Comment (flaymeariver) [reply de-DE/Settings.js]: This is quite a deep topic in the German language, short answer: yes

Long answer: It depends on how accepted an anglicism is to the German language. Generally it is regarded to just rather hypenate the word connection, if you are unsure. In this particular case, session is a regularly used term in IT, but it isn't accepted enough into the German language to regard it as a native word. This distinction is a bit harder in the word Team-Management where even German dictionaries seem to argue about the correct way to spell it. Therefore I went for the safer approach to hyphenate it. If you're interested in the theory, this article explains it kind of nice: https://www.scribbr.de/wissenschaftliches-schreiben/zusammengesetzte-anglizismen/ (would need Google Translate I suppose)

File: de-DE/index.js (hunk 4 above)
Comment (flaymeariver) [reply de-DE/index.js]: done

File: de-DE/Settings.js (hunk 3 above)
Comment (NoelDavies) [reply de-DE/Settings.js]: Dude as long as you're happy with it:P

Comment (flaymeariver) [reply de-DE/Settings.js]: haha sorry, just my notorious curiousity 😄

**Review by NoelDavies — APPROVED** [review]

**Pull request merged**